context/auth.tsx

```diff
@@ -1,16 +1,44 @@
 import React from "react"
 import { loadFirebase } from "./firebase"
 import axios from "../util/axios"
+import firebase from "firebase"
+import { ProfileSerializer } from '../types/backend';
+import { AxiosResponse } from "axios";
 
-const AuthContext = React.createContext({})
+export type userType = firebase.User | null;
```

Comment:
We can directly export this type from `firebase.js`

Reply:
done